sunpy/coordinates/tests/test_utils.py

```diff
@@ -19,6 +20,9 @@
 # calculate these coordinates using the inner angle formulae as listed here:
 # https://en.wikipedia.org/wiki/Great-circle_distance
 
+combined_pattern = (re.escape("One dimensional numpy ndarrays only") + "|" + re.escape("All value in points array must be strictly >=0 and <=1.") + "|" + re.escape("Incorrectly specified \"points\" keyword value.")
```

Comment:
Shrink this down to maybe either the first or second element?

Reply:
I think it may cause errors, I used another variable here because skipping any of the above caused issues and made the match sub-string very verbose.

Reply:
If the test is raising different messages, they should be matched to the exact one.

Reply:
Noted. Done. Thank You.

Reply:
Also another tip, you do not need to leave a comment after you fix everything, you can just mark the conversation as resolved.